packages/utilities/projects/ag-grid/src/scss/theme.scss

```diff
@@ -0,0 +1,252 @@
+@use '../../../utilities/dist' as *;
```

Comment:
The `indeterminate` state of the checkbox is not visible:

<img width="1344" alt="Bildschirmfoto 2024-10-02 um 16 59 06" src="https://github.com/user-attachments/assets/5bfd171a-1146-47ad-a45a-0541baab1379">

See: https://www.ag-grid.com/angular-data-grid/row-selection-multi-row/